shell/creators/pkg/init

```diff
@@ -18,6 +18,7 @@ const files = [
 const topLevelScripts = {
   'build-pkg':  './node_modules/@rancher/shell/scripts/build-pkg.sh',
   'serve-pkgs': './node_modules/@rancher/shell/scripts/serve-pkgs',
+  'publish-pkgs': './node_modules/@rancher/shell/extension/scripts/publish',
```

Comment:
shouldn't this be `./node_modules/@rancher/shell/scripts/extension/publish` instead of `./node_modules/@rancher/shell/extension/scripts/publish` ?

Reply:
@aalves08 Yes, you're right. Fixed